### ISSUE TITLE
Entraînement de XGBoost sur 2022.

### ISSUE THREAD

**Issue by danielaDaliesio** [linhnguyen0132/IFT3710_Projet#4]
Entraîner le modèle XGBoost sur les données de 2022 en prenant en note les métriques de top-30 error ainsi que top-1 accuracy. De plus,
prendre en note les train/val loss pour voir l'évolution du modèle.
